recipe/meta.yaml

```diff
@@ -15,20 +15,22 @@ build:
     - ceci-update-for-version-2 = ceci.update_pipelines_for_ceci_2:main
     - ceci-ancestors = ceci.tools.ancestors:main
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 2
+  noarch: python
+  number: 3
 
 requirements:
   host:
-    - python
+    - python {{ python_min }}
     - setuptools >=61.0
     - wheel
     - setuptools-scm >=6.2
     - pip
   run:
-    - python
+    - python >={{ python_min }}
+    - setuptools
     - pyyaml >=5.1
     - psutil
-    - graphlib-backport  # [py<39]
+    - graphlib-backport
```

Comment:
Since `python_min=3.9` this shouldn't be needed at all.

Reply:
This line still needs to be removed.  Also, I have no review powers in this repo.